proto/cosmos/slashing/v1beta1/tx.proto

```diff
@@ -6,12 +6,21 @@ option (gogoproto.equal_all) = true;
 
 import "gogoproto/gogo.proto";
 
-// MsgUnjail is an sdk.Msg used for unjailing a jailed validator, thus returning
-// them into the bonded validator set, so they can begin receiving provisions
-// and rewards again.
+// Msg defines the slashing Msg service.
+service Msg {
+  // Unjail defines a method for unjailing a jailed validator, thus returning
+  // them into the bonded validator set, so they can begin receiving provisions
+  // and rewards again.
+  rpc Unjail(MsgUnjail) returns (MsgUnjailResponse);
+}
+
+// MsgUnjail defines the Msg/Unjail request type
 message MsgUnjail {
   option (gogoproto.goproto_getters)  = false;
   option (gogoproto.goproto_stringer) = true;
 
   string validator_addr = 1 [(gogoproto.moretags) = "yaml:\"address\"", (gogoproto.jsontag) = "address"];
-}
\ No newline at end of file
+}
+
+// MsgUnjailResponse defines the Msg/Unjail response type
+message MsgUnjailResponse {}
```

Comment:
how about having a "global" type for empty message, eg: `EmptyResponse`. We should reuse it instead of creating an empty response for each module.

Reply:
`Empty` already exists but there's a reason we don't use it. See the buf style guide: https://buf.build/docs/style-guide